pkg/kubelet/util/store/filestore_test.go

```diff
@@ -70,20 +80,20 @@ func TestFileStore(t *testing.T) {
 
 	// Test add data.
 	for _, c := range testCases {
-		_, err = store.Read(c.key)
+		t.Log("test case: ", c)
+		_, err := store.Read(c.key)
 		assert.Error(t, err)
 
 		err = store.Write(c.key, []byte(c.data))
 		if c.expectErr {
 			assert.Error(t, err)
 			continue
-		} else {
-			assert.NoError(t, err)
 		}
 
+		require.NoError(t, err)
 		// Test read data by key.
 		data, err := store.Read(c.key)
-		assert.NoError(t, err)
+		require.NoError(t, err)
```

Comment:
I found out that assert helper libs are against the Go style guide, but we can change these tests in a future PR.